runhouse/main.py

```diff
@@ -430,7 +431,7 @@ def cluster_down(
 
     Example:
 
-        ``$ runhouse cluster down /sashab/rh-basic-cpu``
+        ``$ runhouse cluster down rh-basic-cpu``
```

Comment:
Maybe we should add `console.print("Cluster is terminated.")` just for consistency?

Reply:
for this one we already log in the teardown APIs, so it will be a bit repetitive i think